packages/amplication-data-service-generator/src/server/resource/service/service.base.template.ts

```diff
@@ -1,10 +1,10 @@
-import { PrismaService } from "nestjs-prisma";
+import { PrismaService } from "../../prisma/prisma.service";
```

Comment:
it would be nice if we'll create alias to pPisma in our generated code instead of import it as it is now

Reply:
We will create a new PR and add `webpack.config` file to handle this

Reply:
This was resolved by using `ts.config.json`, providing a "paths"